package.json

```diff
@@ -10,8 +10,9 @@
     "benchmark:ssr": "cd benchmarks && yarn ssr",
     "benchmark:stylesheet": "cd benchmarks && yarn stylesheet",
     "benchmark:module-traversal": "cd packages/babel-plugin && yarn benchmark:module-traversal",
-    "clean": "rm -rf node_modules/.cache && rm -rf test/dead-code-elimination/dist && rm -f test/dead-code-elimination/tsconfig.tsbuildinfo && workspaces-run -- rm -rf dist -- rm -f tsconfig.tsbuildinfo -- rm -f tsconfig.browser.tsbuildinfo",
+    "clean": "rm -rf node_modules/.cache && rm -rf test/dead-code-elimination/dist && rm -f test/dead-code-elimination/tsconfig.tsbuildinfo && workspaces-run -- rm -rf dist -- rm -f tsconfig.tsbuildinfo -- rm -f tsconfig.browser.tsbuildinfo -- rm -rf build",
     "start": "yarn build:main && npx nodemon --exec \"start-storybook -p 6006 --ci\" --watch packages/babel-plugin/ -e tsx",
+    "start:ssr": "cd playground/ssr && yarn start",
```

Comment:
This requires `yarn build` otherwise it will throw error if someone runs `yarn clean` and then this command.

Reply:
cool i'll add it